BlackJack/index.js

```diff
@@ -29,189 +41,295 @@ makeCard();
-playerCards.appendChild(playerImg)
-
-let playerRandomCard2 = Math.floor(Math.random() * allCard.length) +1;
-playerImg2.src = `cards/${allCard[playerRandomCard2]}`
-playerCards.appendChild(playerImg2)
+playerImg.src = `cards/${allCard[playerRandomCard]}`;
+playerCards.appendChild(playerImg);
 
+let playerRandomCard2 = Math.floor(Math.random() * allCard.length) + 1;
+playerImg2.src = `cards/${allCard[playerRandomCard2]}`;
+playerCards.appendChild(playerImg2);
 
 //save player first card value
-let firstCardN = allCard[playerRandomCard]
-let secondCardN = allCard[playerRandomCard2]
-let firstCardNum = firstCardN.charAt(0)
-let secondCardNum = secondCardN.charAt(0)
+let firstCardN = allCard[playerRandomCard];
+let secondCardN = allCard[playerRandomCard2];
+let firstCardNum = firstCardN.charAt(0);
+let secondCardNum = secondCardN.charAt(0);
 
-if(firstCardNum + secondCardNum === '1A' || firstCardNum + secondCardNum === 'JA' || firstCardNum + secondCardNum === 'QA' || firstCardNum + secondCardNum === 'KA' ){
-  alert('BlackJack You Win')
+if (
+  firstCardNum + secondCardNum === "1A" ||
+  firstCardNum + secondCardNum === "JA" ||
+  firstCardNum + secondCardNum === "QA" ||
+  firstCardNum + secondCardNum === "KA"
+) {
+  alert("BlackJack You Win");
   location.reload();
-}else if(firstCardNum + secondCardNum === 'A1' || firstCardNum + secondCardNum === 'AJ' || firstCardNum + secondCardNum === 'AQ' || firstCardNum + secondCardNum === 'AQ' ){
-  alert('BlackJack You Win')
+} else if (
+  firstCardNum + secondCardNum === "A1" ||
+  firstCardNum + secondCardNum === "AJ" ||
+  firstCardNum + secondCardNum === "AQ" ||
+  firstCardNum + secondCardNum === "AQ"
+) {
+  alert("BlackJack You Win");
   location.reload();
 }
 
-function firstCardNumFunc (){
-  let firstCardNum = firstCardN.charAt(0)
-  if(firstCardNum === '2' || firstCardNum === '3' || firstCardNum === '4' || firstCardNum === '5' || firstCardNum === '6' || firstCardNum === '7' || firstCardNum === '8' || firstCardNum === '9'){
-     return firstCardN = firstCardNum;
-  }else if( firstCardNum === 'J' || firstCardNum === 'K' || firstCardNum === 'Q' || firstCardNum === '1'){
-    return firstCardN = '10'
-  }else if(firstCardNum === 'A'){
-  return firstCardN = '1'
+function firstCardNumFunc() {
+  let firstCardNum = firstCardN.charAt(0);
+  if (
+    firstCardNum === "2" ||
+    firstCardNum === "3" ||
+    firstCardNum === "4" ||
+    firstCardNum === "5" ||
+    firstCardNum === "6" ||
+    firstCardNum === "7" ||
+    firstCardNum === "8" ||
+    firstCardNum === "9"
+  ) {
+    return (firstCardN = firstCardNum);
+  } else if (
+    firstCardNum === "J" ||
+    firstCardNum === "K" ||
+    firstCardNum === "Q" ||
+    firstCardNum === "1"
+  ) {
+    return (firstCardN = "10");
+  } else if (firstCardNum === "A") {
+    return (firstCardN = "1");
   }
 }
-firstCardNumFunc ()
-console.log(firstCardN)
+firstCardNumFunc();
+console.log(firstCardN);
```

Comment:
Another thing to improve is the use of function returned values. In this case you're using the function to modify and update variables. which works, but a function should refrain from updating variables outside of it's scope. It should only return the new values. and then assign the returned values to the global variables